app/app.js

```diff
@@ -125,6 +116,27 @@ module.exports = (options) => {
     res.render('component-preview', { bodyClasses, previewLayout })
   })
 
+  // All components view
+  app.get('/examples/all-components', function (req, res, next) {
+    const components = fileHelper.allComponents
+
+    res.locals.componentData = components.map(componentName => {
+      let componentData = fileHelper.getComponentData(componentName)
+      let firstExample = componentData.examples[0]
```

Comment:
This is assuming that the _first_ example will be the 'default' example – to be consistent with what we do elsewhere can we re-use the logic from [here](https://github.com/alphagov/govuk-frontend/blob/61db9b3cec7249c72337cfd12f3c045876fa10e2/app/app.js#L90-L96) to pull out whichever example has the name 'default'?

Reply:
I suppose I was not too concerned that this logic does get the default (even though it does by coincidence).

Let's go with your suggestion and have the same intent as elsewhere.